code/datums/martial/earthquake.dm

```diff
@@ -0,0 +1,152 @@
+	return ..()
+
+/datum/martial_art/earthquake/on_projectile_hit(mob/living/carbon/human/A, obj/projectile/P, def_zone)
+	. = ..()
+	if(A.incapacitated(FALSE, TRUE)) //NO STUN
+		return BULLET_ACT_HIT
+	if(!(A.mobility_flags & MOBILITY_USE)) //NO UNABLE TO USE
+		return BULLET_ACT_HIT
+	if(A.dna && A.dna.check_mutation(HULK)) //NO HULK
+		return BULLET_ACT_HIT
+	if(!isturf(A.loc))
+		return BULLET_ACT_HIT
+	if(A.in_throw_mode)
+		A.visible_message("<span class='danger'>[A] reflects the projectile with their rock-hard abs!</span>", "<span class='userdanger'>You peform the metamorphic technique!</span>")
+		playsound(get_turf(A), pick('sound/weapons/bulletflyby.ogg', 'sound/weapons/bulletflyby2.ogg', 'sound/weapons/bulletflyby3.ogg'), 75, TRUE)
+		P.firer = A
+		P.setAngle(rand(0, 360))//SHING
+		return BULLET_ACT_FORCE_PIERCE
+	return BULLET_ACT_HIT
+
+/datum/martial_art/earthquake/teach(mob/living/carbon/human/H, make_temporary = FALSE)
+	. = ..()
+	if(!.)
+		return
+	to_chat(H, "<span class='userdanger'>Your gauntlets harden into place. You recall the forgotten art of [name]!</span>")
+	to_chat(H, "<span class='danger'>Check the status bar to learn the basics of EQS.</span>")
+	ADD_TRAIT(H, TRAIT_NOGUNS, EARTHQUAKE_TRAIT)
+	ADD_TRAIT(H, TRAIT_PIERCEIMMUNE, EARTHQUAKE_TRAIT)
+	ADD_TRAIT(H, TRAIT_STUNRESISTANCE, EARTHQUAKE_TRAIT)
+	ADD_TRAIT(H, TRAIT_IGNOREDAMAGESLOWDOWN, EARTHQUAKE_TRAIT)//you are slow as fuck, but your speed is constant
+	H.physiology.brute_mod *= 0.4 //the earth is strong
+	H.physiology.burn_mod *= 0.4 //burn the earth and it becomes even sterner in strength
+	H.physiology.stamina_mod *= 2.0 //the earth is slow and difficult to carry topple those who use it(this combines with slowdown to become a clear PVP weakness- use it!)
+	H.physiology.stun_mod *= 0.5 //the earth's existing movement cannot be so easily stopped, though
+	H.physiology.pressure_mod *= 0.1 //rocks don't die in space dummy
+	H.physiology.cold_mod *= 0.3 //cold mods are different to burn mods, they do stack however
+	H.physiology.heat_mod *= 0.3 //earth can get pretty toasty, but it always melts eventually
+	H.physiology.tox_mod *= 1.5//pollution is bad, kids
+
+/datum/martial_art/earthquake/on_remove(mob/living/carbon/human/H)
+	. = ..()
+	to_chat(H, "<span class='userdanger'>Your connection to the earth fades. You have lost knowlege of [name]...</span>")
+	REMOVE_TRAIT(H, TRAIT_NOGUNS, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_PIERCEIMMUNE, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_STUNRESISTANCE, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_IGNOREDAMAGESLOWDOWN, EARTHQUAKE_TRAIT)
+	H.physiology.brute_mod = initial(H.physiology.brute_mod)
+	H.physiology.burn_mod = initial(H.physiology.burn_mod)
+	H.physiology.stamina_mod = initial(H.physiology.stamina_mod)
+	H.physiology.stun_mod = initial(H.physiology.stun_mod)
+	H.physiology.pressure_mod = initial(H.physiology.pressure_mod)
+	H.physiology.cold_mod = initial(H.physiology.cold_mod)
```

Comment:
you forgot to also set heat_mod to intial

Reply:
I'll fix it RQ, but note this is a WIP version and the proper implementation of EQS will likely be part of my port of newtg's martial art framework